src/accordion/accordion.js

```diff
@@ -79,7 +79,9 @@ angular.module('ui.bootstrap.accordion', ['ui.bootstrap.collapse'])
     link: function(scope, element, attrs, accordionCtrl) {
       accordionCtrl.addGroup(scope);
 
+      scope.openClass = attrs.openClass || 'panel-open';
       scope.$watch('isOpen', function(value) {
+        element.toggleClass(scope.openClass, value);
```

Comment:
Any DOM manipulation can be done to change the class of the element - in that case, relying on toggle to correctly toggle the class would be completely incorrect, since it makes a poor assumption that the actual class reflects the state of `isOpen`. The coupling of the model value and the DOM value is not strong enough to make this assumption in a library.